godtossing/Pos/Ssc/GodTossing/DB.hs

```diff
@@ -35,12 +37,11 @@ gtGlobalStateToBatch :: GtGlobalState -> GtOp
 gtGlobalStateToBatch = PutGlobalState
 
 initGtDB :: MonadDB m => VssCertificatesMap -> m ()
-initGtDB _ = pass
--- Commented due to CSL-1113, maybe uncomment when we will use store serialization.
-  --   whenNothingM_ (gsGetBi @_ @GtGlobalState gtKey) $
-  --       gsPutBi gtKey (def {_gsVssCertificates = vcd})
-  -- where
-  --   vcd = VCD.fromList . toList $ certs
+initGtDB certs =
+    whenNothingM_ (gsGetBi @_ @GtGlobalState gtKey) $
+        gsPutBi gtKey (def {_gsVssCertificates = vcd})
+  where
+    vcd = VCD.fromList . toList $ certs
```

Comment:
1. Given that this code is touched, I would remove `certs` parameter, because it's always constant.
2. It should be checked for performance regression. It was commented out for a reason and maybe it's not ok to just uncomment it.